### ISSUE TITLE
Skeleton is not displayed

### ISSUE THREAD

**Issue by Ma-Kn** [WonderMediaProductions/Maya2glTF#36]
Hello!

I tried to display it with gltf but the skeleton
Do not show.
I do not know how to do it

![image](https://user-images.githubusercontent.com/2511656/42377583-ce744814-815e-11e8-9410-2ee83782f5fa.png)

**Comment by Ziriax** [comment]
What viewer are you using? It's best to ask this question to the developers of that viewer.

Closing this for now, if this turns out to be a bug with Maya2glTF, I will re-open it.

**Comment by Ziriax** [comment]
That is indeed interesting, I will re-open this issue.

**Comment by Ziriax** [comment]
I guess you can't share your model with me?

**Comment by Ziriax** [comment]
This seems to be a bug in the viewer related to the way Maya2glTF handles segment scale compensation.

I reported it here: https://github.com/donmccurdy/three-gltf-viewer/issues/101

You have two options: [disable segment scale compensation on all your joints](http://3.bp.blogspot.com/-4F1JTEOEyy4/T2Q7apc20-I/AAAAAAAAADY/CJS6wX7eDH0/s1600/SegmentScaleCompensation.jpg), or add `-isc` in the `extra flags` text fields in the Maya2glTF exporter UI.

Note that if you want to perform non-uniform scaling on any of your joints, disabling segment scale compensation might give strange animation blending results (depending on the engine you are using).

Closing this, since it is indeed unrelated to Maya2glTF.
